internal/resources/grafana/resource_report.go

```diff
@@ -356,20 +356,20 @@ func DeleteReport(ctx context.Context, d *schema.ResourceData, meta interface{})
 	return diag
 }
 
-func schemaToReport(d *schema.ResourceData) (models.CreateOrUpdateConfigCmd, error) {
+func schemaToReport(d *schema.ResourceData) (models.CreateOrUpdateReportConfig, error) {
```

Comment:
Why `models.CreateOrUpdateReportConfig` and not `models.CreateOrUpdateConfigCmd`. Just curious because it worked 😅.

Reply:
It's from the update to the client here: https://github.com/grafana/grafana-openapi-client-go/pull/51. Changed here in grafana: https://github.com/grafana/grafana-enterprise/pull/6003